tests/ansible-network.vyos/defaults/main.yml

```diff
@@ -1,2 +1,3 @@
 ---
-# defaults file for ansible-network.vyos
\ No newline at end of file
+# defaults file for ansible-network.vyos
+
```

Comment:
Remove this line (and same in other files)

Reply:
So, I'll remove the new lines I just added? Or the comments?